src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -859,4 +859,10 @@ Valid from:</comment>
     <value>Some framework assembly reference TFMs are missing a platform version: {0}</value>
     <comment>0 - comma-separated list of reference group TFMs</comment>
   </data>
+  <data name="ZipFileTimeStampeModified" xml:space="preserve">
+    <value>{0} file lastwritetime(modified) timestamp changed from {1} to {2} due to zip file doesn't support timestamp before 01/01/1980 limitation.</value>
```

Comment:
```suggestion
    <value>Last write timestamp for '{0}' changed from '{1}' to '{2}' because the zip file format does not support timestamp values before 01/01/1980.</value>
```

Reply:
Ok. Changed to below, now it has upper limit too.

![image](https://user-images.githubusercontent.com/8766776/102397993-89d34980-3f93-11eb-8442-4c79a49f73ab.png)